jscomp/core/js_exp_make.ml

```diff
@@ -1265,16 +1265,22 @@ let bigint_comp (cmp : Lam_compat.comparison) ?comment (e0: t) (e1: t) =
   bin ?comment (Lam_compile_util.jsop_of_comp cmp) e0 e1
 
 let bigint_div ~checked ?comment (e0: t) (e1: t) =
-  if checked then
-    runtime_call Js_runtime_modules.bigint "div" [e0; e1]
-  else
-    bigint_op ?comment Div e0 e1
+  match e1.expression_desc with
+  | Number (Bigint { i = i1 }) when i1 = "0" || i1 = "-0" -> 
```

Comment:
Not sure about this change.
What if `e1` is a variable and not a constant: should't the runtime to a check for division by zero?

Reply:
Correct! How about this? https://github.com/rescript-lang/rescript-compiler/pull/6670/commits/7d22f9cebc05449469976fefb24ea39652f940ea